pkg/signup/signup_handler.go

```diff
@@ -5,22 +5,31 @@ import (
 	"log"
 	"net/http"
 
-	"github.com/codeready-toolchain/registration-service/pkg/configuration"
 	"github.com/gin-gonic/gin"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
```

Comment:
this import should be in the "second" section, ie, above the `github.com/gin-gonic/gin`, to follow our way of sorting/organizing imports

Reply:
mmh..I always had it vice versa, the most generic on top, then the more specific and the local, most specific stuff on the bottom..

Reply:
Fixed 5f83a27.